doc/source/project/releases.rst

```diff
@@ -0,0 +1,70 @@
+======================
+Galaxy Release Process
+======================
+
+This document outlines the Galaxy release cycle so administrators and users know
+when to expect updates and what each update includes, making it easier to plan
+for upgrades and maintain their systems.  It does not cover the actual execution
+of the release process, which is covered elsewhere. (link)
+
+
+Major or Long Term Support (LTS) Releases
+-----------------------------------------
+
+The first release of each year is Galaxy's annual LTS release. The LTS release
+is a major version update corresponding to the year, for example, 24.0.  This
+release receives significant and thorough testing by a dedicated team that does
+end-to-end testing of a variety of analysis protocols and will be supported and
```

Comment:
```suggestion
release receives significant and thorough end-to-end testing by a dedicated team and will be supported and
```

Reply:
Or better, "thorough testing", i.e., no "end-to-end". I'd keep it open.

Reply:
@jdavcs Please feel free to push whatever tweaks you want